### ISSUE TITLE
User is created as admin if attribute is set to false in data bag

### ISSUE THREAD

**Issue by yoshiwaan** [hw-cookbooks/chef-server-populator#33]
Hi,

The current logic check for making a user in the client recipe incorrectly creates a user as an admin even if the admin attribute is set to false. 
This is because the current logic looks in a different place for the admin key than what is specified in the examples:
```
    item.merge('client' => data_bag_item(node[:chef_server_populator][:databag], bag_item)['id'],
      'pub_key' => item['client_key'],
      'enabled' => item['enabled'],
      'admin' => item.fetch('admin', true),
      'password' => item.fetch('password', SecureRandom.urlsafe_base64(23)),
      'orgs' => item.fetch('orgs', {}))
    ...
    execute "set user org: #{item['client']}" do
      command "chef-server-ctl org-user-add #{item['org']} #{item['client']} #{'--admin' if item['admin']}"
    end
```
In the data bag the admin flag is set under the org:
```
{
  "id": "<user>",
    "orgs": {
      "<org>": {
        "enabled": true,
        "admin": false  <---
      }
    }
  }
}
```
Either the examples need updating or the code does, but you take away the ability to have a user be an admin in one org but not another if you don't update the code.

**Comment by luckymike** [comment]
@yoshiwaan thanks for raising this. Can you confirm that you've seen this occur with a user data bag item that has `"admin": false` explicitly set in the org hash?

In the user creation section, we're setting `item[:admin] = options[:admin]` which I expected to supersede the setting from above. That said, with this refactor, clients should default to `"admin": false` anyway, so that needs changing, which will expose any issues with that user block.

Unfortunately, as chef-server-ctl does not currently have any commands for querying the admins, I can't write a test for this, but I'll fiddle with things to manually to ensure the behavior is correct.

**Comment by luckymike** [comment]
@yoshiwaan I was able to duplicate the issue, fix coming!

**Comment by luckymike** [comment]
This should be resolved in version 1.1.4. Please note that we don't yet support user creation across multiple orgs. That enhancement should come in the next round of updates.

**Comment by yoshiwaan** [comment]
Yes, I can confirm that I saw the --admin flag passed when a user was
created with "admin" : false attribute nested under org
